docs/unified-test-documentation/dasharo-security/201-verified-boot.md

```diff
@@ -487,3 +487,57 @@ command:
 
 After pressing `ENTER` the DUT should immediately move to the next stages of
 booting.
+
+## VBO011.001 Booting after flashing with valid binary (Ubuntu 22.04)
+
+**Test description**
+
+This test aims to verify whether after flashing the DUT with the valid binary,
+the DUT will boot correctly from the default slot and no recovery popup will be
+displayed.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+1. `OPERATING_SYSTEM` = Ubuntu 22.04
+
+**Test setup**
+
+1. Proceed with the
+   [Test cases common documentation](#test-cases-common-documentation) section.
+1. Download `cbmem` and `flashrom` from <https://cloud.3mdeb.com/index.php/s/zTqkJQdNtJDo5Nd>
+   to the DUT.
+1. Disable Secure Boot.
+1. Slot A is flashed with an binary with wrong vboot keys as in test case
+   `VBO008.001`.
```

Comment:
This point suggest, that we need to do the test. Couldn't we limit ourselves to saying that the device is currently being flashed with a binary with the wrong keys?

Reply:
Sure, done